docs/index.md

```diff
@@ -43,88 +43,61 @@ Mouse includes enrichments for:
 
 #### Example:
 
-```scala
-scala> import mouse.all._
+```scala mdoc
 import mouse.all._
 
-scala> true.option("Its true!")
-res0: Option[String] = Some(Its true!)
```

Comment:
It would be nice to have those things as comments. WDYT?

Reply:
Sorry, I don't understand?

Reply:
https://github.com/typelevel/mouse/pull/305#discussion_r794435277